### DISCUSSION TITLE
Finding Electronic Supplementary Information (ESI) for OKane2022 paper

### DISCUSSION THREAD

**Discussion by HassanSewailem** [pybamm-team/PyBaMM#2662] [Q&A]
Dear PyBamm team, 

While I was reading the "Lithium-ion battery degradation: how to model it" paper, I noticed ESI was mentioned, which had model equations and parameters that could be useful. 

I looked for it online, but I could not find it.  Do you happen to have a link for the ESI for this paper? 

Thank you in advance for your time and consideration.

Best regards,
Hassan Sewailem

**Comment by brosaplanella** [comment]
The ESI should be available on the [journal website](https://pubs.rsc.org/en/content/articlelanding/2022/CP/d2cp00417h).
